jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -858,14 +858,11 @@ public void run()
             if (LOG.isDebugEnabled())
                 LOG.debug("Runner started for {}", QueuedThreadPool.this);
 
-            Runnable job = null;
+            // Start with fake NOOP job so we go idle on first iteration
+            Runnable job = NOOP;
```

Comment:
Are these changes to QTP intended to be in this PR?

Reply:
oops - good catch thanks

Reply:
I agree with @lachlan-roberts, these changes seem out of place.